homework/hw37/main.js

```diff
@@ -0,0 +1,76 @@
+const postContainer = document.getElementById('post-container');
+const postTitle = document.getElementById('post-title');
+const postBody = document.getElementById('post-body');
+const commentsBtn = document.getElementById('comments-btn');
+const commentsContainer = document.getElementById('comments-container');
+const commentsList = document.getElementById('comments-list');
+const errorContainer = document.getElementById('error-container');
+const errorMessage = document.getElementById('error-message');
+const searchBtn = document.getElementById('search-btn');
+const postIdInput = document.getElementById('post-id-input');
```

Comment:
In the future try to avoid the such amount of elements in global scope, you all the time keep them in a memory

Some selectors you can write in a place of usage in local scope
```
function displayPost() {
  const postContainer = document.getElementById('post-container');
  const postTitle = document.getElementById('post-title');
  const postBody = document.getElementById('post-body');
  
  // ...
}
```